### PR TITLE
feat/ovos_conf from ovos_utils

### PR DESCRIPTION
**Pull request by NeonJarbas**
logic moved to ovos_utils https://github.com/OpenVoiceOS/ovos_utils/pull/12

### CODE REVIEW

File: mycroft/skills/msm_wrapper.py [hunk 1]
```diff
@@ -27,14 +27,15 @@
...
-from mycroft.util.file_utils import get_temp_path
-from mycroft.configuration import BASE_FOLDER
 
 from mock_msm import \
     MycroftSkillsManager as MockMSM, \
     SkillRepo as MockSkillRepo
 
+from ovos_utils.configuration import get_xdg_base
+from mycroft.util.file_utils import get_temp_path
+from mycroft.util.log import LOG
```
Comment (NeonDaniel) [inline mycroft/skills/msm_wrapper.py:37]: Should this import from ovos?

File: mycroft/skills/settings.py [hunk 2]
```diff
@@ -249,7 +250,7 @@ def upload(self):
             LOG.debug('settingsmeta.json not uploaded - device is not paired')
 
         if not synced and not self._stopped:
-            self.upload_timer = Timer(ONE_MINUTE, self.upload)
+            self.upload_timer = Timer(60, self.upload)
```
Comment (NeonDaniel) [inline mycroft/skills/settings.py:253]: Should ONE_MINUTE variable be deprecated?

File: test/unittests/lock/test_lock.py [hunk 3]
```diff
@@ -22,33 +22,33 @@
 
 from mycroft.lock import Lock
 from mycroft.util.file_utils import get_temp_path
-from mycroft.configuration import BASE_FOLDER
```
Comment (NeonDaniel) [inline test/unittests/lock/test_lock.py:25]: Does `BASE_FOLDER` need to stay for backwards compat?

**Review by NeonDaniel — APPROVED** [review]
Did not test; left some comments for consideration

File: test/unittests/lock/test_lock.py (hunk 3 above)
Comment (JarbasAl) [reply test/unittests/lock/test_lock.py:25]: no, this does not exist in mycroft-core so it's fine

**Pull request merged**